test/common/upstream/health_check_fuzz_test.cc

```diff
@@ -15,6 +15,12 @@ DEFINE_PROTO_FUZZER(const test::common::upstream::HealthCheckTestCase input) {
     return;
   }
 
+  if (input.health_check_config().health_checker_case() ==
```

Comment:
Why not add this to: `HealthCheckFuzz::initializeAndReplay`? (and still return immediately)

Reply:
Oh yeah! The original problem was the default case would break then still hit initialize and replay. Switched to return. Thanks!